instance_control/bubuku_cluster.py

```diff
@@ -0,0 +1,198 @@
+import json
+import time
+from subprocess import call
+
+import boto3
+import click
+from instance_control import create_instance
+
+
+@click.group()
+def cli():
+    pass
+
+
+def stop_kafka(ip: str, user: str, odd: str):
+    piu = ["piu", ip, "-O", odd, "\"detaching ebs, terminate and launch instance\""]
+    call(piu)
+    ssh = ["ssh", "-tA", user + '@' + odd, "ssh", "-o", "StrictHostKeyChecking=no", user + '@' + ip,
+           "'docker stop taupageapp'"]
+    call(ssh)
+
+
+def terminate_instance(instance):
+    instance.terminate()
+
+    print("Instance is {}. Waiting ...".format(instance.state['Name']))
+    while True:
+        instance.load()
+        if instance.state['Name'] == 'terminated':
+            print("Instance is terminated")
+            return
+        time.sleep(10)
+
+
+def delete_alarm(cluster_config: dict, instance):
+    cw_client = boto3.client('cloudwatch')
+    cw_client.delete_alarms(
+        AlarmNames=['{}-{}-auto-recover'.format(cluster_config['cluster_name'], instance.instance_id)])
+
+
+def wait_volumes_attached(ec2_client, ec2_resource):
+    response = ec2_client.describe_volumes(Filters=[{'Name': 'tag:Name', 'Values': ['kafka-logs-ebs']}])
+    volumes = [ec2_resource.Volume(v['VolumeId']) for v in response['Volumes']]
+    print('Wait for {} volume(s) to be attached'.format(len(volumes)))
```

Comment:
I think that logging of every step is very, very important task. And probably it is better to use python logging library for that